<launch>
	<arg name="yolo_config_file" 		default="/home/hatem/darknet/yolo4/darknet/traffic-lights-4/yolov4-bosch.cfg" />
	<arg name="yolo_weights_file" 		default="/home/hatem/darknet/yolo4/darknet/yolov4-bosch_10000.weights" />		
	
	<arg name="detection_threshold" 	default="0.41" />	
	
	<arg name="camera_id" default="/"/>
    <arg name="camera_info_src" default="/camera_info"/>
    <arg name="image_src" default="/image_raw"/>
	
	<node pkg="op_tlr" name="op_tlr" type="op_tlr" output="screen">
       	<param name="yolo_config_file" 				value="$(arg yolo_config_file)" />
       	<param name="yolo_weights_file" 			value="$(arg yolo_weights_file)" />       	
       	
       	<param name="detection_threshold" 			value="$(arg detection_threshold)" />       	
       	       	
       	<param name="image_src" 					value="$(arg image_src)"/>
    	<param name="camera_frame" 					value="$(arg camera_id)"/>
    	<remap from="camera_info" to="$(arg camera_id)$(arg camera_info_src)"/>
       	            
    </node>
</launch>
